packages/editor/src/extensions/code-block/tests/code-block.test.ts

```diff
@@ -191,3 +192,28 @@ test("editing code in a highlighted code block should not be too slow", async ()
   expect(timings.reduce((a, b) => a + b) / timings.length).toBeLessThan(16);
   expect(editorElement.outerHTML).toMatchSnapshot();
 });
+
+test("Switching codeblock language should register the new language", async () => {
```

Comment:
Are we just checking that language got registered? Shouldn't we also check that the code also got highlighted?

Reply:
The problem we solved here was the languages were not being registered which was the root cause of the code not being hghlighted so that is why I made the test to only check for new language registeration